### PR TITLE
Hex code

### PR DESCRIPTION
**Pull request by MridulGupta007**
HEX CODE GENERATOR

Creates random color codes and also provides their respective CSS Code which can be pasted in the CSS file.
Has a reset button to restore default color( white ).

### CODE REVIEW

**Comment by Rishikavishnoi** [comment]
@MridulGupta007 mention issue no. and add a screenshot  of your project in the README.md file

File: Projects/Hex Code Generator/Readme.md [hunk 1]
```diff
@@ -2,3 +2,9 @@ This is a Hex Code Generator
 
 The purpose is to generate random colors and their respective CSS code for developers to use;
 
+
+Tech Stack used: HTML, CSS, JavaScript
+
+
+Issue number - 124
+
```
Comment (MridulGupta007) [inline Projects/Hex Code Generator/Readme.md:10]: Screenshot (222)

**Review by Vanshiii203 — CHANGES_REQUESTED** [review]
@MridulGupta007 Attach a screenshot of your project in the Readme.md file and do not mention the issue in the readme file, link it with the PR (by using #issuenumber)

**Comment by MridulGupta007** [comment]
closes Issue #174

**Comment by MridulGupta007** [comment]
Screenshot (222)

**Comment by Vanshiii203** [comment]
@MridulGupta007 Please remove the issue number from readme.md file

**Comment by MridulGupta007** [comment]
Done

**Review by Vanshiii203 — APPROVED** [review]

**Pull request merged**